### DISCUSSION TITLE
waterfox classic missing in ubuntu 22.04 repo

### DISCUSSION THREAD

**Discussion by eadmaster** [hawkeye116477/waterfox-deb-rpm-arch-AppImage#53] [Q&A] [answered]
it was present in prev versions:

https://download.opensuse.org/repositories/home:/hawkeye116477:/waterfox/xUbuntu_22.04/

**Comment by hawkeye116477** [comment, marked as answer]
There was 1 small problem, but got solved. It's now available.

**Comment by eadmaster** [comment]
thanks, it is working fine.
